docs/UserGuide.md

```diff
@@ -367,87 +367,82 @@ Examples:
 
 ### Adding an event: `jeren`
```

Comment:
my bad didn't say properly, remember to replace the `jeren` section with `<command-name>`:
eg. Adding an event: `addEvent`

Reply:
same with all the subsequent lines containing `jeren`

Reply:
changed